src/main.ts

```diff
@@ -694,6 +694,20 @@ export class Purchases {
     return this._appUserId;
   }
 
+  /**
+   * Sets attributes for the current user. Attributes are useful for storing additional, structured information on a customer that can be used elsewhere in the system.
+   * For example, you could store your customer's email address or additional system identifiers through the applicable reserved attributes, or store arbitrary facts like onboarding survey responses, feature usage, or other dimensions as custom attributes.
```

Comment:
Not sure if it's worth saying here that we won't cache nor retry sending these attributes... I'm a bit concerned with the difference in behavior in the hybrids making it more difficult to use there... I think something I will want to discuss with the SDK team.

Reply:
Added a note